tests/fireproof/fireproof.test.ts

```diff
@@ -84,7 +97,7 @@ describe("public API", function () {
   });
   it("should be a database instance", function () {
     expect(db).toBeTruthy();
-    expect(db instanceof Database).toBeTruthy();
+    expect(isDatabase(db)).toBeTruthy();
```

Comment:
not sure our downstream users can be expected not to use `instanceof` maybe we should have both assertions? I'm not sure what isDatabase would be used for outside of this, so maybe we remove the test?